styleguide/_themes/global/scss/_fonts.scss

```diff
@@ -55,6 +55,9 @@ a {
     }
 }
 
+ul li, ol li{
+  text-align: left;
```

Comment:
Why do all list items need to be left aligned? Are they being centered by something else?